dynawo/sources/Models/Modelica/PreassembledModels/CMakeLists.txt

```diff
@@ -11,6 +11,17 @@
 # Preassembled models
 
 set(MODELICA_PREASSEMBLED_MODELS
+    ACVoltageControl.xml
+    ACEmulation.xml
+    DCVoltageControl.xml
+    DCVoltageControlSide.xml
+    ActivePowerControl.xml
+    ActivePowerControlSide.xml
+    LimitsCalculationFunction.xml
+    BlockingFunction.xml
+    GeneralBlockingFunction.xml
+    DCLine.xml
+    HVDCStandard.xml
```

Comment:
Do you need a preassembled model for all of them?

Reply:
I was also wondering but you already corrected. Indeed, the idea is that any model that could be used to represent a component should have a preassembled model. Thus it should only be the final models somehow.